src/lib.rs

```diff
@@ -62,3 +63,14 @@ pub struct FsBackendDesc {
     pub mounted_time: DateTime<Local>,
     pub config: Option<serde_json::Value>,
 }
+
+pub fn makedev(major: u64, minor: u64) -> dev_t {
```

Comment:
Please move it to `utils/src/compact.rs` where we can maintain these cross-target helpers.